doc/nrf/gs_programming.rst

```diff
@@ -87,6 +87,28 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
    .. build_SES_projimport_start
 
 4. Click :guilabel:`OK` to add the project to SES.
+
+Activating license for first time
```

Comment:
```suggestion
Activating a license for the first time
```

Could we make it more of a full sentence? I think adding in these articles makes it sound more professional.